src/monitor/pgautofailover.sql

```diff
@@ -97,6 +97,7 @@ CREATE TABLE pgautofailover.node
     groupid              int not null,
     nodename             text not null,
     nodeport             int not null,
+    sysidentifier        bigint,
```

Comment:
Can we add a simple CHECK constraint here that this can only be `null` whenever the `reportedstate` is `init` or something? In all other cases this should never be `NULL` afaict.

Reply:
Done.